.github/workflows/pull-request-automation.yml

```diff
@@ -29,6 +29,10 @@ jobs:
             - name: Setup Node.js and install dependencies
               uses: ./.github/setup-node
 
+            - name: Npm build
+              # TODO: We don't have to build the entire project, just the action itself.
+              run: npm run build:packages
```

Comment:
Let us complete the TODO in this PR

- Rename the `lib` directory in `packages/project-management-automation` to `src`
- Update `include` in `packages/project-management-automation/tsconfig.json` to `"src"`
- Then, instead of building all the packages, we can run `node bin/packages/build.js packages/project-management-automation/src/**/*.ts`
- We can also update the name here to say "Build project automation package" instead of "Npm build"